object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,8 @@
+/* exported setValue */
+function setValue(object, key, value) {
+  object[key] = value;
+  if (Object.keys(object).length !== 0) {
+    return object;
+  }
+  return undefined;
```

Comment:
This can be removed

Reply:
are you talking about the condition in the `if` statement specifically?